r/tests/testthat/test-compressed.R

```diff
@@ -27,6 +27,7 @@ if (identical(Sys.getenv("APPVEYOR"), "True")) {
   test_that("Compression codecs are included in the Windows build", {
```

Comment:
We don't have any R builds on AppVeyor, do we? The condition should be that the test is running on Windows instead.

Reply:
Good catch! these tests probably haven't been running. I don't see a good way to check if a particular test ran in the CI. I'll double check on my local machine tomorrow.

Reply:
I've fixed the condition and moved it into a `skip_if()` call, which allows us to see in CI output whether it was skipped.